app/controllers/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.valid?
+      @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+    @user_temp = User.new
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @user_temp = User.new(user_params)
+
+    if @user_temp.valid?
```

Comment:
Use `if @user.valid?` instead.

Reply:
Thanks for the suggestion. It's been fixed